ansible/roles/base-buster/meta/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+dependencies:
+  - role: track_etc_directory
```

Comment:
Maybe this is a bit too much indirection? Can we put the content of this in the role directly for the time being and abstract it when N > 1 roles need to depend on this and not the base one?

Reply:
Moved to base-buster and dom0-bootstrap